aspnetcore/fundamentals/servers/kestrel.md

```diff
@@ -629,6 +629,20 @@ webBuilder.ConfigureKestrel(serverOptions =>
 });
 ```
 
+## Connection logging
+
+Call <xref:Microsoft.AspNetCore.Hosting.ListenOptionsConnectionLoggingExtensions.UseConnectionLogging*> to emit verbose logs for byte-level communication on a connection. Connection logging is helpful for troubleshooting problems in low-level communication, such as during TLS encryption and behind proxies.
```

Comment:
```suggestion
Call <xref:Microsoft.AspNetCore.Hosting.ListenOptionsConnectionLoggingExtensions.UseConnectionLogging*> to emit Debug level logs for byte-level communication on a connection. Connection logging is helpful for troubleshooting problems in low-level communication, such as during TLS encryption and behind proxies. If UseConnectionLogging is placed before UseHttps it will log the encrypted traffic. If it's placed after it will log the decrypted traffic.
```